File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int n=nums.length;
        int cur=0;
        int ans=0;
        for(int i=0;i<n;i++){
            if(nums[i] == 1){
                cur+=1;
                ans=Math.max(cur,ans);
            }else{
                cur=0;
            }
        }
        return ans;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        ans = 0
        for i in nums :
            if i == 1 :
                count += 1 
            else :
                if count > ans :
                    ans = count
                count = 0
        if count > ans:
            return count
        return ans